JSDOMParser.js

```diff
@@ -452,6 +487,23 @@
     this.value = value;
   };
 
+  Attribute.prototype = {
+    setValue: function(newValue) {
```

Comment:
I don't want to do that, because before this, consumers assigned directly into `attr.value`. That shouldn't happen because then `this._decodedValue` isn't deleted and will have a stale value.